Miguel Tuñón/prueb_suma.c

```diff
@@ -0,0 +1,49 @@
+#include<stdio.h>
+#define NF 3
+void main ()

```

Comment:
No se que hacias en este problema, pero esto no calcula la suma de la diagonal de la matriz. Debias en un ciclo for luego de leida la matriz, cuando i==j guardar el valor en una variable suma.